### ISSUE TITLE
Support 2bit reference sequence

### ISSUE THREAD

**Issue by kammoji** [GMOD/Apollo#1872]
Dear all,

Tried to run ./apollo run-local 8085 on multiple versions of Apollo: 2.0.7; 2.0.8; git clone version
All sets out nicely. When I try to add an annotation to the User-created annotations area, an error is raised:

"
| Error 2018-03-15 14:28:18,938 [Actor Thread 4] ERROR apollo.AnnotationEditorController  - CAUGHT ERROR through websocket call: groovy.lang.GroovyRuntimeException: Ambiguous method overloading for method java.lang.Integer#div.
Cannot resolve which method to invoke for [null] due to overlapping prototypes between:
        [class java.lang.Character]
        [class java.lang.Number]
| Error 2018-03-15 14:28:18,940 [Actor Thread 4] ERROR apollo.AnnotationEditorController  - exception groovy.lang.GroovyRuntimeException: Ambiguous method overloading for method java.lang.Integer#div.
Cannot resolve which method to invoke for [null] due to overlapping prototypes between:
        [class java.lang.Character]
        [class java.lang.Number]
| Error 2018-03-15 14:28:18,940 [Actor Thread 4] ERROR apollo.AnnotationEditorController  - exception message Ambiguous method overloading for method java.lang.Integer#div.
Cannot resolve which method to invoke for [null] due to overlapping prototypes between:
        [class java.lang.Character]
        [class java.lang.Number]
| Error 2018-03-15 14:28:18,941 [Actor Thread 4] ERROR apollo.AnnotationEditorController  - username ***************@helsinki.fi
| Error 2018-03-15 14:28:18,941 [Actor Thread 4] ERROR apollo.AnnotationEditorController  - error destination message: /topic/AnnotationNotification/user/***************@helsinki.fi
"
![apollo_2018](https://user-images.githubusercontent.com/22050587/37463835-1979476e-285f-11e8-86f2-0fc278e967c9.png)

I tried a combination of Apollo versions mentioned above and JBrowse versions 1.12.4-dev; 1.12.5-dev and 1.13.0-dev. All produce the same error

**Comment by nathandunn** [comment]
Usually it means that field isn’t being found in the data that its trying to annotate with, but we can see.

1 - does it happen for all annotations in the track?
2 - does it happen for other tracks?
3 - Can you post what you see when you get view details from one of the failing features?
4 - if you can post the message sent in the web socket frame that would be helpful (open up the developer tools, click on the network tab, and the WS tab, reload page and attempt doing an annotation and you should see a “SEND” message in the frame). 
5 - if you could post your JSON data that you uploaded that would be great (or email to the nathandunn@lbl.gov)

Thanks,

Nathan

**Comment by kammoji** [comment]
Dear Nathan,

Thanks for your message. I was able to solve this by changing the prepare-refseqs reference type from 2bit to FASTA (as the features are mapped to FASTA).

As to your questions I found out that:

1. Yes, it happens to all annotations in the track
2. Maybe, if they're mapped to same FASTA. Did not have the time to test.
3. Details were looking what I expected, I am reluctant to post them here as this is unpublished work.
4. I used Firefox browser and I could not see a SEND message in the WS tab in developer tools (only a GET message)
5. Reluctant to do this for the same reason as in 3.

Thanks anyway,
Juhana K
